easybuild/easyblocks/l/lammps.py

```diff
@@ -676,15 +713,53 @@ def sanity_check_step(self, *args, **kwargs):
             for check_file in sanity_check_test_inputs
         ]
 
+        # add accelerator-specific tests
+        if self.pkg_intel:  # INTEL package
+            custom_commands.append(
+                'from lammps import lammps; l=lammps(cmdargs=["-sf", "intel"]); l.file("%s")' %
+                os.path.join(self.installdir, "examples", "msst", "in.msst")
+            )
+        if self.cfg['kokkos']:  # KOKKOS package
+            if self.cuda:  # NOTE: requires a GPU to run
```

Comment:
This is a difficult requirement to meet, we should be able to support a cross-compilation. In EESSI, for example, this is going to cause problems as we must cross-compile for a lot of architectures. You could make it conditional on the `nvidia-smi` command running successfully?

Reply:
See for example 
https://github.com/easybuilders/easybuild-easyblocks/blob/6a0f356cda727feaaf79fdb72ae2cfbe48bbdf28/easybuild/easyblocks/t/tensorflow.py#L1048-L1054